front/src/common/Map/Search/MapSearchStation.tsx

```diff
@@ -120,32 +143,64 @@ const MapSearchStation = ({ updateExtViewport, closeMapSearchPopUp }: MapSearchS
     closeMapSearchPopUp();
   };
 
-  const clearSearchResult = () => {
-    setSearch('');
-    resetSearchResult();
-    setSearchResults(undefined);
-    dispatch(updateMapSearchMarker(undefined));
-  };
-
   return (
     <>
       <div className="d-flex mb-2">
-        <span className="flex-grow-1">
+        <div className="flex-fill">
           <InputSNCF
-            type="text"
-            placeholder={t('map-search:placeholdername')}
             id="map-search-station"
+            name="map-search-station"
+            placeholder={t('map-search:placeholdername')}
```

Comment:
As there are now 2 more inputs on the same line, the placeholder of this one doesn't fit anymore. We should use a title (and add a props for that in InputSNCF) so the user can see it in full.
```suggestion
            placeholder={t('map-search:placeholdername')}
            title={t('map-search:placeholdername')}
```